apps/portal/lib/graphql/resolvers/OldEvent.resolver.ts

```diff
@@ -25,25 +26,33 @@ export default class OldEventResolver {
   ): Promise<EventCheckin[]> {
     const event_data = await this.firebaseService.returnEventsbyProfile(netId, email);
 
-    const events = await this.eventService.findOldEventID(event_data);
+    if (event_data.length != 0) {
```

Comment:
Instead of doing something like this: 
```js
if (something) {
   doThingOne();
   doThingTwo();
} else {
   return [];
}
```

Consider doing this: 
```js
if (!something) { 
   return [];
}

doThingOne();
doThingTwo();
```

This way, less nesting will be involved, thus making code cleaner in general.